components/Hero.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import {
+  Dimensions,
+  StyleSheet,
+  Text,
+  View,
+  Image,
+  ImageBackground,
+  Touchable,
+  TouchableOpacity,
+} from 'react-native';
+
+const Hero = ({ article }) => {
+  return (
+    <>
+      <TouchableOpacity>
+        {article && (
```

Comment:
Why do you check for article here and not in regular Article?

Reply:
yes, so what will happen if you dont have any articles? Hero artcle have this condition to handle it, but Article doesn't. You should add it there too I, think

Reply:
Perhaps we can get rid of it now actually when it's inside the renderItem?

Reply:
na messes up the initial render of the hero if we do so

Reply:
yes, it's fixed now.